openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -153,7 +153,7 @@ The Wallet MAY send one Batch Credential Request to the Batch Credential Endpoin
 * multiple Credentials of the same type/doctype bound to different proofs, or
 * multiple Credentials of different types/doctypes bound to different proofs.
 
-In the course of the authorization process, the Credential Issuer MAY also request Credential presentation as a means to authenticate or identify the End-User during the issuance flow, as illustrated in a use case in (#use-case-2).
+In the course of the authorization process, the Credential Issuer MAY also request Credential presentation as a means to authenticate or identify the End-User during the issuance flow, as illustrated in (#use-case-5).
```

Comment:
```suggestion
In the course of the authorization process, the Credential Issuer MAY also request Credential presentation as a means to authenticate the End-User during the issuance flow, as illustrated in (#use-case-5).
```

I would not say "Identify", even if I fully understand the meaning, because the identification process (id proofing) is something that happens during the provisioning of a digital identity, while the issuance of a digital credential gives an verifiable instance of the digital credential

that's why I love to distinguish identification (id proofing and eid provisioning), issuance (DC provisioning) and authentication (use of the DC for getting authenticated)

Reply:
no hard feelings about this. wording on idenfitication/authentication is always hard. what do others think?

Reply:
I just copied the existing text. I think you may use credentials to authenticate even if SIOP is probably advised

Reply:
> I just copied the existing text.

 I know. I would like to stick to this text.